File: 4A.js
var x = readline();

if (x < 4) {
  print("NO");
} else {
  print(x % 2 === 0 ? "YES" : "NO");
}

// Alternative

const evenDivide = (inp) => {
  if (inp % 2 !== 0) {
    return "NO";
  }
  for (let i = 2; i <= inp / 2; i += 2) {
    if ((inp - i) % 2 === 0) {
      return "YES";
    }
  }
  return "NO";
};

console.log(evenDivide(4));

File: 4A.py
def evenDivide(inp):

    if inp%2 != 0:
        return 'No'
    for i in range(2, inp, 2):
        if i*2 == inp:
            return 'Yes'
        for j in range(i+2, inp, 2):
            if i + j == inp:
                return 'Yes'
    return 'No'
print(evenDivide(int(input())))

# Alternative  -> Better Soultion

def evenDivide(inp):
    if inp < 4 or inp % 2 != 0:
        return "NO"
    else:
        for i in range(2, inp, 2):
            if (inp - i) % 2 == 0:
                return "YES"
    return "NO"

print(evenDivide(int(input())))